### PR TITLE
Added "beta-confluent-kafka-zookeeper" directory to services, update broken img link/typos

### PR DESCRIPTION
**Pull request by yishan-lin**
## Description
https://jira.mesosphere.com/browse/DCOS-20642

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.11/monitoring/logging/access-component-logs/index.md [hunk 1]
```diff
@@ -8,40 +8,38 @@ beta: true
...
 
-1.  Select **Organization** and choose **Users**. Select an existing or create a new user. 
-    
+1.  Select **Organization** and choose **Users**. Select an existing or create a new user.
+
     ![New user](/1.11/img/new-user-generic.png)
-    
-1.  From the **Permissions** tab, select **ADD PERMISSION**.
 
-    ![Add permission to user](/1.11/img/permission-user.png)
```
Comment (joerg84) [inline pages/1.11/monitoring/logging/access-component-logs/index.md]: it seems like the picture is functional in the 1.10 version, would it make sense to copy it from the 1.10 over here? @yishan-lin

Comment (yishan-lin) [reply pages/1.11/monitoring/logging/access-component-logs/index.md]: 👍

File: pages/1.11/monitoring/logging/access-component-logs/index.md [hunk 2]
```diff
@@ -29,7 +29,9 @@ Here is the [permission](/1.11/security/ent/perms-reference/) that is required t
     ![New user](/1.11/img/new-user-generic.png)
 
 1.  From the **Permissions** tab, select **ADD PERMISSION**.
-    
+
+    ![Add permission to user](/1.10/img/permission-user.png)
```
Comment (yishan-lin) [inline pages/1.11/monitoring/logging/access-component-logs/index.md]: @joerg84 Added the 1.10 link, should work now vs. removing it entirely.  Leaving it to you to merge. 👍

File: pages/1.11/deploying-services/index.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ enterprise: false
 
 DC/OS uses Marathon to manage processes and services. Marathon is the "init system" for DC/OS. Marathon starts and monitors your applications and services, automatically healing failures.
 
-A native Marathon instance is installed as a part of DC/OS installation. After DC/OS has started, you can manage the native Marathon instance through the **Services** tab of the DC/OS web interface or from the DC/OS CLI with the `dcos marathon` command.
```
Comment (yishan-lin) [inline pages/1.11/deploying-services/index.md]: If you go to your Services tab under 1.11, it says "no running services" - It's not manageable from there.

**Pull request merged**